lib/search/ui/widgets/search_bar.dart

```diff
@@ -0,0 +1,207 @@
+import 'dart:async';
+
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/data/enums/search_result_type.dart';
+import 'package:fluttercon/common/data/models/search_result.dart';
+import 'package:fluttercon/common/utils/misc.dart';
+import 'package:fluttercon/common/utils/router.dart';
+import 'package:fluttercon/common/widgets/resolved_image.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import 'package:fluttercon/l10n/l10n.dart';
+import 'package:fluttercon/search/cubit/search_cubit.dart';
+import 'package:fluttercon/search/cubit/search_state.dart';
+import 'package:go_router/go_router.dart';
+
+class SearchBarWidget extends StatelessWidget {
+  const SearchBarWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const SearchBarView();
+  }
```

Comment:
This looks unnecessary. You can copy over all the widget code from `SearchBarView` into this build method.

Reply:
Fixed